### ISSUE TITLE
Snake Breed Identification

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#228]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Snake Breed Identification</br>
:red_circle: **Aim** : You have been hired as a Deep Learning engineer to create a sophisticated model that can detect the breed of a snake from its image.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/oossiiris/hackerearth-deep-learning-identify-the-snake-breed</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by rafiya2003** [comment]
I would like to work on this, frankly I don't know much but I will learn and do. I am a newbie(first year UG student). I want to learn and grow

**Comment by Abhinavcode13** [comment]
Please assign this issue to me @abhisheks008 I have good knowledge in machine learning and python libraries like NumPy pandas etc

**Comment by abhisheks008** [comment]
> Please assign this issue to me @abhisheks008 I have good knowledge in machine learning and python libraries like NumPy pandas etc

Comment according to the issue template.
@Abhinavcode13

**Comment by Sagar201003** [comment]
i can even , work on this issue , as I have made classification projects . So I can work on this . Pls assign this to me .

**Comment by Vansh912** [comment]
Full name : Vansh Tiwari
GitHub Profile Link : https://github.com/Vansh912
Email ID : vanshtiwari.912@gmail.com
Participant ID (if applicable):
Approach for this Project : will start by exploring the dataset and do preprocessing according to the requirement, will use pca,random forest ,svm 
What is your participant role? (Mention the Open Source program): SSOC 23

**Comment by abhisheks008** [comment]
This project repository demands projects based on Deep Learning methods, not ML methods. Try to implement DL methods for solving this issue.

**Comment by Praneesh-Sharma** [comment]
Full name : Praneesh Sharma
GitHub Profile Link : https://github.com/Praneesh-Sharma
Email ID : pegasus30042004@gmail.com
Participant ID (if applicable):
Approach for this Project : 
    - will start by implementing custom CNN models
    - will implement pretrained models like YOLO or Vision Transformer 
    - compare the results and decide on which model to implement
What is your participant role? (Mention the Open Source program) SSOC'23

**Comment by abhisheks008** [comment]
@Vansh912 I am waiting for your approach, else this issue will be assigned to someone else.

**Comment by Vansh912** [comment]
Sorry for the late response I am not that good in DL but know image processing models like alexnet,googlenet,inception i can try using different models to classify them

**Comment by abhisheks008** [comment]
It will be better to learn those things and then try to solve this kind of issues.

Anyways issue assigned to you @Praneesh-Sharma

**Comment by pawaspy** [comment]
Full name : Pawas Pandey
GitHub Profile Link : github.com/pawaspy
Email ID : pawaspy2633@gmail.com
Participant ID (if applicable): NA
Approach for this Project : I will create a sequential model and train the model and test on the data.
What is your participant role? Contributor
Assign me this

**Comment by abhisheks008** [comment]
Can you mention specifically, what are the deep learning models you want to use for this project?
@pawaspy

**Comment by pawaspy** [comment]
I will use mobile v2 net and cnn

**Comment by abhisheks008** [comment]
Are you part of the CodePeak 2023 event?

**Comment by pawaspy** [comment]
No, how can I join this event?
I really want to work on this project

**Comment by abhisheks008** [comment]
Hi @pawaspy you can check out the CodePeak website for further details about the event. Here is the website, https://www.codepeak.tech/

**Comment by pawaspy** [comment]
I have joined the code peak. Can you assign me now this task?

**Comment by the-silent-geek** [comment]
Full name : D. Uday Kiran
GitHub Profile Link : https://github.com/the-silent-geek
Email ID : udaykir3911@gmail.com
Approach for this Project : This can be implemented by using pre-trained models like vgg16, ResNet, and DenseNet. And would like to try vision transformer approach.
What is your participant role? social winter of code 23

**Comment by abhisheks008** [comment]
> Full name : D. Uday Kiran GitHub Profile Link : https://github.com/the-silent-geek Email ID : [udaykir3911@gmail.com](mailto:udaykir3911@gmail.com) Approach for this Project : This can be implemented by using pre-trained models like vgg16, ResNet, and DenseNet. And would like to try vision transformer approach. What is your participant role? social winter of code 23

One issue at a time. Please finish your previous issue first.

**Comment by the-silent-geek** [comment]
Could you assign me this isssue now.

**Comment by the-silent-geek** [comment]
Full name : D. Uday Kiran
GitHub Profile Link : https://github.com/the-silent-geek
Email ID : udaykir3911@gmail.com
Approach for this Project : This can be implemented by using pre-trained models like vgg16, ResNet, and DenseNet. And would like to try vision transformer approach.
What is your participant role? social winter of code 23

**Comment by abhisheks008** [comment]
Issue assigned to you @the-silent-geek
